capella2polarion/__main__.py

```diff
@@ -22,22 +22,34 @@
 
 
 @click.group()
-@click.option("--debug", is_flag=True, default=False)
+@click.option(
+    "--debug", is_flag=True, envvar="CAPELLA2POLARION_DEBUG", default=False
+)
 @click.option(
     "--polarion-project-id",
     type=str,
     required=False,
     default=None,
-    envvar="POLARION_PROJECT_ID",
+    envvar="CAPELLA2POLARION_PROJECT_ID",
 )
 @click.option(
     "--polarion-url",
     envvar="POLARION_HOST",
     type=str,
 )
 @click.option("--polarion-pat", envvar="POLARION_PAT", type=str)
-@click.option("--polarion-delete-work-items", is_flag=True, default=False)
-@click.option("--capella-model", type=cli_helpers.ModelCLI(), default=None)
+@click.option(
+    "--polarion-delete-work-items",
+    is_flag=True,
+    envvar="CAPELLA2POLARION_DELETE_WORK_ITEMS",
+    default=False,
+)
+@click.option(
+    "--capella-model",
+    type=cli_helpers.ModelCLI(),
+    envvar="CAPELLA2POLARION_CAPELLA_MODEL",
+    default=None,
+)
```

Comment:
This will lead to not being able to look into --help for any command. We have good logging messages for the cases when there is no PAT, HOST and Project ID provided. I would keep it like this.

Reply:
You still can see `--help`, as long as you provide all the globally required options (which I admit is a bit inconvenient and counter-intuitive, but this is how click works). And it feels a bit weird to do so much custom validation after the fact when all of this can be done by click before even launching our code.

> We have good logging messages for the cases when there is no PAT, HOST and Project ID provided.

My main point was that these messages are missing entirely for `--capella-model`, both for the `synchronize` and `render-documents` subcommands.